### DISCUSSION TITLE
Description List

### DISCUSSION THREAD

**Discussion by jeffreylauwers** [nl-design-system/backlog#216] [Component suggestions]
<img width="480" alt="visuele weergave van het description list component" src="https://github.com/nl-design-system/backlog/assets/545207/2b393d27-7276-4422-b6ab-e8cb8a62e7d3">

We willen graag meer leren over het **Description List** component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook of screenshots. Met deze informatie kunnen we bewijzen dat het component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

[Link naar Slack bericht](https://codefornl.slack.com/archives/C7KDQRXMZ/p1706799159341149?thread_ts=1706798708.688949&cid=C7KDQRXMZ)

---

## Naam
Description List

## Doel
Toont een lijst aan termen en hun beschrijving, bijvoorbeeld een woordenlijst.

## Gerelateerde componenten
[Data Summary](https://github.com/orgs/nl-design-system/discussions/356), [Form Summary](https://github.com/orgs/nl-design-system/discussions/215), [Table](https://github.com/orgs/nl-design-system/discussions/294)

## Onderzoek
[The Description List element - MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl)
`<dl>` (description list) HTML element zoals beschreven in MDN docs.

## Richtlijnen

**WCAG**
https://www.w3.org/WAI/WCAG22/Understanding/unusual-words.html

**Comment by jeffreylauwers** [comment]
## Tilburg

- [Figma ontwerp](https://www.figma.com/file/wM6Clh8a0Ak2P8GIyJ0NKg/Publicatie-portaal---Gemeente-Tilburg?type=design&node-id=946%3A63624&mode=design&t=LhhVBkOdun0lr3tA-1)

---

### Screenshots / Varianten

**Figma**

![Scherm­afbeelding 2024-05-14 om 12 43 11](https://github.com/nl-design-system/backlog/assets/545207/921c4f74-5cc5-43c0-a028-da0a2f9c883b)

---

### Notities
- Binnen Drawer na klik op 'Begrippenlijst'.
- Lijkt ook doorzoekbaar.

**Comment by Robbert** [comment]
# Toegankelijkheidsverklaringen.nl

<img width="786" alt="Screenshot 2024-07-04 at 09 49 19" src="https://github.com/nl-design-system/backlog/assets/30694/ad947d27-55c9-4aad-817b-5a6a7e6f389e">

## Notities

- Gebruikt `<dl>`
- Niet een zelfstandig component, dit is de `status-explanation__list`

**Comment by jeffreylauwers** [comment]
Aram Limpens (Gemeente Amsterdam) stelde in een draadje van de [Slack oproep voor goedkeuring](https://codefornl.slack.com/archives/C7KDQRXMZ/p1719294727641529) de volgende vraag:

> Is er een reden om voor Definition List te gaan ipv Description List? [De HTML spec gebruikt 'description'](https://html.spec.whatwg.org/multipage/grouping-content.html#the-dl-element)

Het kernteam is het daar mee eens heeft besloten om vanaf nu over Description list te spreken in plaats van Definition list.

**Comment by jeffreylauwers** [comment]
## ✨ Dit component is nu Help Wanted ✨

Help je mee hem door de Community stappen te krijgen?
[Description list op het Community bord](https://github.com/orgs/nl-design-system/projects/29/views/1?filterQuery=description+list)

**Comment by jeffreylauwers** [comment]
Tijdens de Open Dag van september de naamgeving van de componenten Summary List en Description List besproken. Uiteindelijk zijn we uitgekomen op [Form Summary](https://github.com/orgs/nl-design-system/discussions/215), [Data Summary](https://github.com/orgs/nl-design-system/discussions/356) en Description List.

**Comment by jeffreylauwers** [comment]
## Amsterdam

Zojuist gevonden bij Amsterdam. Voor nu alleen even het linkje:

https://designsystem.amsterdam/?path=/docs/components-text-description-list--docs

**Comment by eslook** [comment]
## ✨ Dit component is nu Community ✨

Help je mee hem door de Candidate stappen te krijgen?
[Description List op het Candidate bord](https://github.com/orgs/nl-design-system/projects/80/views/1?filterQuery=Description+List)
